src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -25,7 +25,7 @@ public function assertResponseOk()
 	 */
 	public function assertResponseStatus($code)
 	{
-		return PHPUnit::assertEquals($code, $this->response->getStatusCode());
+		return PHPUnit::assertEquals($code, $this->response->getStatusCode(), "Expected status code $code, got $actual");
```

Comment:
Fixed. @GrahamCampbell could you please merge this by squashing it in the process? Authorship isn't really important for such a minor enhancement. Thanks.